web/src/features/panels/zone/ShareButton.tsx

```diff
@@ -44,23 +44,16 @@ export function ShareButton({
       reference.current?.publish();
     };
 
-    if (hasMobileUserAgent && (await CapShare.canShare())) {
-      share(
-        {
-          title: 'Electricity Maps',
-          text: 'Check this out!',
-          url: shareUrl,
-        },
-        toastMessageCallback
-      ).then((shareCompleted) => {
-        if (shareCompleted) {
-          trackShareCompletion();
-        }
-      });
-    } else {
-      copyToClipboard(shareUrl, toastMessageCallback);
-    }
-    trackShareClick();
+    await (hasMobileUserAgent && (await CapShare.canShare())
```

Comment:
In my opinion this is more readable as an if statement. Now a reader is required to know the precedence of `await` vs the ternary operator. It's also a little strange to have a single await applied to multiple branches.

Snce we only care about the side effects here, I think an imperative statement more accurately describes the intent of the code.

Reply:
Same but our prettier rules force it to be ternary 🤦

Reply:
I should've known! 😧

Reply:
I'll approve this but I might look into changing Prettier when I have some down time...